Dockerfiles/midstream/unit_tests.py

```diff
@@ -0,0 +1,41 @@
+
+import unittest
+import os
+import subprocess
+
+
+class Testing(unittest.TestCase):
+
+    def test_postive(self):
+        exec_cmd = """
+        /run_tests.py
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        self.assertEqual(0, result.returncode)
+
+    def test_negative_extract_operator_bundle(self):
+        exec_cmd = """
+        export UMOCI_BIN_PATH=/root
+        /run_tests.py
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        self.assertEqual(55, result.returncode)
+
+    def test_negative_validate_operator_bundle(self):
+        exec_cmd = """
+        export WORK_DIR=/kandel
```

Comment:
Please document why we should fail with return code 60 here.
Is it because of wrong work directory ?

Reply:
Yes exactly, I will create comments per unit test so it is clear.